pkg/sysinfo/cgroup2_linux.go

```diff
@@ -66,6 +69,29 @@ func newV2(quiet bool, opts *opts) *SysInfo {
 	return sysInfo
 }
 
+func getSwapLimit() bool {
+	groups, err := cgroups.ParseCgroupFile("/proc/self/cgroup")
+	if err != nil {
+		return false
+	}
+
+	g, found := groups[""]
+	if !found {
+		return false
+	}
+
+	g = path.Clean(g)
+	if g == "." || g == "/" {
```

Comment:
- How can g be "."?
- When g is "/", we are in a cgroup namespace, so we should check /sys/fs/cgroup/memory.swap.max

Reply:
- If g is empty, clean colapses it to ".". I understand empty would be some kind of a bug, so collapsing to "." should be treated as a bug
- Regarding second comment, wasn't sure. I will remove that one.

Reply:
To make things more visible I could first check if g == "" and then do a Clean. I think "" is the only case when Clean collapses to "."